webapp/login/actions.js

```diff
@@ -5,9 +5,12 @@ import * as CognitoAuth from '../app/cognitoAuth'
 import * as LoginState from './loginState'
 import { hideAppLoader, initUser, showAppLoader, showNotificationMessage } from '../app/actions'
 
+export const emailUpdate = 'login/email/update'
```

Comment:
i would follow the convention so far used: calling this type loginEmailUpdate

Reply:
right, changed